### DISCUSSION TITLE
Concurrent remote write simulation to Thanos Receiver

### DISCUSSION THREAD

**Discussion by vgoudphilips** [thanos-io/thanos#4460] [Questions & Answers]
Hi All,

Need your help in simulating below use cases.

We are trying to simulate the concurrent load on the Thanos receiver from various prometheus sources to do remote write.

Usecase: If I want to send 200 concurrent remote write connections to thanos receiver.

Is there any option to simulate via tool or utility to send the telemetry data to Thanos receiver without installing Prometheus and configuring remote write.

Thank you for your help.

**Comment by yeya24** [comment]
There are many options like:
- https://github.com/open-fresh/avalanche
- https://github.com/observatorium/up
- https://github.com/m3dbx/promremotebench

If you are using kubernetes you can also try https://github.com/thanos-io/thanosbench/pull/34.
